state/state.go

```diff
@@ -67,13 +78,23 @@ func loadState(db dbm.DB, key []byte) *State {
 		}
 		// TODO: ensure that buf is completely read.
 	}
+
+	v := s.loadValidators(s.LastBlockHeight)
+	if v != nil {
+		s.lastHeightValidatorsChanged = v.LastHeightChanged
+	} else {
+		s.lastHeightValidatorsChanged = 1
```

Comment:
why `1`? should not this be 0? if validators set was never changed

Reply:
if /block?height=0 includes a validator set change, is /validators?height=0 equal to the genesis state or the new state?

Reply:
height=0 will return an error. 

if /block?height=X includes a val set change, it should only show for /validators?height=X+1